src/index.html

```diff
@@ -0,0 +1,127 @@
+                <label><input type="text" name="phone" placeholder="Phone" class="input form__input"></label>
+                <label><input type="text" name="email" placeholder="Email" class="input form__input"></label>
+                <button class="button form__button">consultation</button>
+            </form>
+        </div>
+    </section>
+    <section class="section">
+            <h2 class="heading section__heading">Testimonials<span class="heading__crown"></span></h2>
+            <div class="testimonials">
+                <div class="person">
+                    <img class="person__photo" src="../im/p1.png" alt="Tim Cook">
+                    <h3 class="person__name">Tim Cook</h3>
+                    <p class="person__position">CEO</p>
+                    <p class="person__text">Nice work, Certificates National Association of Realtors (USA).</p>
+                </div>
+                <div class="person">
+                    <img class="person__photo" src="../im/p2.png" alt="Katherine Adams">
+                    <h3 class="person__name">Katherine Adams</h3>
+                    <p class="person__position">Senior Vice President</p>
+                    <p class="person__text">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart. I’m all alone and blissfully happy in the local edge.</p>
+                </div>
+                <div class="person">
+                    <img class="person__photo" src="../im/p3.png" alt="Jonathan Ive">
+                    <h3 class="person__name">Jonathan Ive</h3>
+                    <p class="person__position">Chief Design Officer</p>
+                    <p class="person__text">When my lovely valley of steam rises and half-day sun is above an impermeable.</p>
+                </div>
+            </div>
+        </section>
+    <section class="section">
+        <h2 class="heading section__heading">Contact Us<span class="heading__crown"></span></h2>
+        <div class="contact">
+            <div class="contact__text">
+                <p class="contact__p"><a href="mailto:Hello@miami.com" class="link">Hello@miami.com</a><br>+1 234 555-55-55</p>
+                <p class="contact__p">400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401</p>
+            </div>
+            <form class="form contact__form">
+                <label><input type="text" name="name" placeholder="Name" class="input form__input"></label>
+                <label><input type="text" name="email" placeholder="Email" class="input form__input"></label>
+                <label><textarea name="Message" placeholder="Message" class="input input__textarea form__input"></textarea></label>
+                <button class="button form__button">write to ust</button>
+            </form>
+        </div>
+    </section>
+</main>
+<footer class="footer">
+    <img src="../im/logo.png" alt="logo" class="logo footer__logo">
+</footer>
+</body>
+</html>
```

Comment:
the document must end with 1 empty line
```suggestion
</html>

```